src/components/controls/index.js

```diff
@@ -31,6 +31,9 @@ const styles = theme => ({
     fontSize: '25px',
     paddingRight: '10px',
     color: theme.colors.white,
+    '@media (max-width: 425px)': {
+      fontSize: p => (p.mobile ? '20px' : undefined),
```

Comment:
I don't get what this media query does. If the `width` is smaller than `425px` set the `fontSize` to `20px` but just if `p.mobile`? Why the `p.mobile` and why `undefined` if not?